src/app/plugins/cointoss.plugin.ts

```diff
@@ -0,0 +1,34 @@
+import { Plugin } from '../../common/plugin';
+import { IContainer, IMessage, ChannelType } from '../../common/types';
+import { MessageEmbed } from 'discord.js';
+
+export class CoinToss extends Plugin {
+  public name: string = 'Coin Toss Plugin';
+  public description: string = 'Ask Lion to toss a coin or if given arguments, choose among the arguments.';
+  public usage: string = '!cointoss || !pick <arg_1> <arg_2> ... <arg_n>';
+  public pluginAlias: string[] = ['pick', 'select', 'cointoss', 'coin', 'toss', 'ct'];
+  public permission: ChannelType = ChannelType.Public;
+  private _embed: MessageEmbed = new MessageEmbed();
+  private _coinImg: string = 'https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/240/apple/285/coin_1fa99.png';
+  private _defaultArgs: string[] = ['Heads', 'Tails'];
+  private _responses: string[] = [];
```

Comment:
If you make this a private property, it will affect the next time you call it, this one doesn't need to be a class constant

Reply:
For clarification, are you talking about _responses on line 14?

Reply:
Yes